source/ada/lsp-ada_documents.adb

```diff
@@ -933,6 +933,9 @@ package body LSP.Ada_Documents is
          Tree         : in out LSP.Messages.DocumentSymbol_Tree)
       is
          Next             : LSP.Messages.DocumentSymbol_Trees.Cursor := Cursor;
+         Dummy            : LSP.Messages.DocumentSymbol_Trees.Cursor;
```

Comment:
Actually, you can use `LSP.Messages.DocumentSymbol_Trees.No_Element` instead of `Dummy`.